Camera.MAUI/Platforms/Android/MauiCameraView.cs

```diff
@@ -17,6 +17,7 @@
 using Android.Renderscripts;
 using RectF = Android.Graphics.RectF;
 using Android.Content.Res;
+using DebugOut = System.Diagnostics.Debug;
```

Comment:
One minor nit here: Since we're already using `System.Diagnostics.Debug` in other places in this file (e.g. lines 445, 448 and 588), please use the `DebugOut` alias also for the other occurrences (for consistency). Or alternatively, leave out the alias and use `System.Diagnostics.Debug` verbosely everywhere.

It seems the alias is necessary in the first place due to some name conflict with the `Debug` method from `Android.Views.ViewGroup`, right?

Reply:
Java has `.Debug` overloads all over.
<img width="614" height="145" alt="Screenshot 2025-09-06 093338" src="https://github.com/user-attachments/assets/ab0d8e93-15d7-4e4c-ab3a-9dbe1a82faf2" />

Reply:
Thanks for taking care of this! 👍